test/cypress/integration/card/confirm.cy.test.js

```diff
@@ -52,9 +48,9 @@ describe('Awaiting auth', () => {
     cy.get('#address-city').type(validPayment.city)
     cy.get('#address-postcode').type(validPayment.postcode)
     cy.get('#email').type(validPayment.email)
-  })
 
-  it('Submitting confirmation should redirect to auth waiting page', () => {
```

Comment:
I think this test is checking the intermediary auth waiting page which redirects to confirm after a wait. There are separate tests for the confirmation page https://github.com/alphagov/pay-frontend/blob/master/test/cypress/integration/card/payment.cy.test.js#LL150C1-L150C1 

Are there any other tests for auth waiting page?

Reply:
I see - this makes sense now.
These are the only tests for the auth waiting page.

I now kept the filename the same.

Please take a look.